src/crypto/tls/handshake_client_tls13.go

```diff
@@ -752,7 +753,7 @@ func (hs *clientHandshakeStateTLS13) abortIfRequired() error {
 	if c.ech.offered && !c.ech.accepted {
 		// If ECH was rejected, then abort the handshake.
 		c.sendAlert(alertECHRequired)
-		return errors.New("tls: ech: rejected")
+		return errors.New("tls: ech: rejected " + string(base64.StdEncoding.EncodeToString(c.ech.retryConfigs)))
```

Comment:
Why do we need to cast? `EncodeToString` returns a `string`, no?

Reply:
Oops, that's right. But I also didn't mean to commit this! Reverted.